teach/templates/clubs/email/create_club_user/body.txt

```diff
@@ -0,0 +1,9 @@
+{{ username }},
```

Comment:
This is what the text has been that was approved before (rather, what is being sent right now). I think here it again makes sense to use the username because username and full_name need not necessarily have a correlation but username and email are definitely related